test/cli/remote-dbt-init/ok-interactive/interaction.txt

```diff
@@ -1,14 +1,14 @@
-< Please enter target name.
-> target1
-
-< Enter a name for a workspace to create:
-> dbt_source
-
 < Please enter Keboola Storage API host, eg. "connection.keboola.com".
 > %%TEST_KBC_STORAGE_API_HOST%%
 
 < Please enter Keboola Storage API token. The value will be hidden.
 > %%TEST_KBC_STORAGE_API_TOKEN%%
 
+< Please enter target name.
+> target1
+
+< Enter a name for a workspace to create:
+> dbt_source
+
```

Comment:
Fixed order to match other operations: 1. get dependencies (ask for host/token), 2. interactive dialogs